lib/components/viewers/route-viewer.js

```diff
@@ -230,7 +230,7 @@ class RouteRow extends PureComponent {
           {isActive && (
             <RouteDetails>
               {route.url
-                ? <a href={route.url} target='_blank'>Route Details</a>
+                ? <a className='routeDetails' href={route.url} target='_blank'>Route Details</a>
```

Comment:
Should that color be the default for all links?

Reply:
These links only have to be darker because of the non-white background they're on.